app/api/currentuser/delete.go

```diff
@@ -0,0 +1,55 @@
+package currentuser
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/loginmodule"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:operation DELETE /current-user users currentUserDeletion
+// ---
+// summary: Delete the current user
+// description: Deletes all information stored in the platform related to the current user
```

Comment:
should describe what is deleted exactly (i.e. content from what table)

Reply:
fixed